app/dashboard/admin/page.js

```diff
@@ -208,7 +208,7 @@ export default function AdminDashboard() {
             }
 
             try {
-                const response = await fetch(REGISTER_OFFICAL_URL, {
+                const response = await fetch(REGISTER_OFFICIAL_URL, {
```

Comment:
Update this in import and `constants.js`.
Please run code and check too. If you had run this would be an unknown or missing import.

Reply:
OK